src/Cooked/MockChain/GenerateTx.hs

```diff
@@ -327,13 +351,14 @@ txSkelToCardanoTx txSkel = do
     (txSkelSigners txSkel)
 
 generateTx ::
-  Fee ->
+  Integer ->
+  Wallet ->
   Set Api.TxOutRef ->
   Emulator.Params ->
   Map Script.DatumHash Script.Datum ->
   Map Api.TxOutRef Api.TxOut ->
   Map Script.ValidatorHash (Script.Versioned Script.Validator) ->
   TxSkel ->
   Either GenerateTxError (Cardano.Tx Cardano.ConwayEra)
```

Comment:
Everything is clear from the parameters names but we could add brief haddock descriptions for the function and parameters

Reply:
Most functions (if not all of them) of the generateTx module do not have Haddock comments. I believe this is because those functions are meants to be used internally solely. We could have a issue to increase our comments coverage but I don't think adding a single comment in that specific PR makes sense. What do you think?

Reply:
I think that, ideally, all functions should have haddock documentation, regardless of whether they are exposed in the public API or not. Those documentation comments are useful to both users and tinkerers/developers who often turn out to be mostly the same kind of people.

However, in this specific PR, I don't think it matters.

And for this specific function, names are self explanatory.

So I'm ok to leave it as is.